org.adempiere.base/src/org/compiere/model/MBankStatementLine.java

```diff
@@ -316,4 +326,9 @@ public boolean isDateConsistentIfUsedForPosting(Timestamp headerDateAcct) {
 		return true;
 	}
 	
+	@Override
+	public MDepositBatch getC_DepositBatch() throws RuntimeException {
+		return new MDepositBatch(getCtx(), getC_DepositBatch_ID(), get_TrxName());

```

Comment:
should be `return getC_DepositBatch_ID() > 0 ? new MDepositBatch(getCtx(), getC_DepositBatch_ID(), get_TrxName()) : null;` instead.

Reply:
Fixed